eva-accession-import/src/main/java/uk/ac/ebi/eva/accession/dbsnp/io/SubmittedVariantsNoHgvsLinkReader.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright 2014-2018 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.ac.ebi.eva.accession.dbsnp.io;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.batch.item.database.JdbcCursorItemReader;
+import org.springframework.jdbc.core.ArgumentPreparedStatementSetter;
+import org.springframework.jdbc.core.PreparedStatementSetter;
+import org.springframework.util.DigestUtils;
+
+import uk.ac.ebi.eva.accession.dbsnp.model.VariantNoHgvsLink;
+
+import javax.sql.DataSource;
+import java.sql.Connection;
+import java.sql.SQLException;
+
+import static uk.ac.ebi.eva.accession.dbsnp.io.SubmittedVariantRowMapper.ALLELES_COLUMN;
+import static uk.ac.ebi.eva.accession.dbsnp.io.SubmittedVariantRowMapper.BATCH_HANDLE_COLUMN;
+import static uk.ac.ebi.eva.accession.dbsnp.io.SubmittedVariantRowMapper.BATCH_NAME_COLUMN;
+import static uk.ac.ebi.eva.accession.dbsnp.io.SubmittedVariantRowMapper.CHROMOSOME_COLUMN;
+import static uk.ac.ebi.eva.accession.dbsnp.io.SubmittedVariantRowMapper.CHROMOSOME_START_COLUMN;
+import static uk.ac.ebi.eva.accession.dbsnp.io.SubmittedVariantRowMapper.CONTIG_NAME_COLUMN;
+import static uk.ac.ebi.eva.accession.dbsnp.io.SubmittedVariantRowMapper.CONTIG_ORIENTATION_COLUMN;
+import static uk.ac.ebi.eva.accession.dbsnp.io.SubmittedVariantRowMapper.CONTIG_START_COLUMN;
+import static uk.ac.ebi.eva.accession.dbsnp.io.SubmittedVariantRowMapper.FREQUENCY_EXIST_COLUMN;
+import static uk.ac.ebi.eva.accession.dbsnp.io.SubmittedVariantRowMapper.GENOTYPE_EXIST_COLUMN;
+import static uk.ac.ebi.eva.accession.dbsnp.io.SubmittedVariantRowMapper.LOAD_ORDER_COLUMN;
+import static uk.ac.ebi.eva.accession.dbsnp.io.SubmittedVariantRowMapper.REFERENCE_COLUMN;
+import static uk.ac.ebi.eva.accession.dbsnp.io.SubmittedVariantRowMapper.SS_CREATE_TIME_COLUMN;
+import static uk.ac.ebi.eva.accession.dbsnp.io.SubmittedVariantRowMapper.TAXONOMY_ID_COLUMN;
+
+public class SubmittedVariantsNoHgvsLinkReader extends JdbcCursorItemReader<VariantNoHgvsLink> {
```

Comment:
Name this the same way as the object to be read, so as per suggested in other comment it would be `SubSnpNoHgvsLinkFieldsReader`.